Hunting Queries/SecurityAlert/AlertsForIP.yaml

```
@@ -0,0 +1,34 @@
+id: 0b520385-6a16-4e6f-ba89-c320d857695f
+name: Alerts related to IP
+description: |
+  'Any Alerts that fired related to a given IpAddress during the range of +6h and -3d'
+requiredDataConnectors:
+  - connectorId: AzureSecurityCenter
+    dataTypes:
+      - SecurityAlert
+  - connectorId: MicrosoftCloudAppSecurity
+    dataTypes:
+      - SecurityAlert
+tactics:
+  - Persistence
+  - Discovery
+  - LateralMovement
+  - Collection
+query: |
+
+  let GetAllAlertsWithIp = (suspiciousEventTime:datetime, v_ipAddress:string){
+  //-3d and +6h as some alerts fire after accumulation of events
+  let v_StartTime = suspiciousEventTime-3d;
+  let v_EndTime = suspiciousEventTime+6h;
+  SecurityAlert
+  | where TimeGenerated between (v_StartTime .. v_EndTime)
+  // expand JSON properties
+  | extend Extprop = parsejson(ExtendedProperties)
+  | extend Computer = toupper(tostring(Extprop["Compromised Host"]))
+  | extend Account =  tolower(tostring(Extprop["User Name"]))
+  | where ExtendedProperties contains v_ipAddress or Entities contains v_ipAddress
+  | project StartTimeUtc = StartTime, EndTimeUtc = EndTime, AlertName, Computer, Account, ExtendedProperties, Entities
+  | extend timestamp = StartTimeUtc, AccountCustomEntity = Account, HostCustomEntity = Computer
```

Suggest adding the IP address parameter
```suggestion
  | extend timestamp = StartTimeUtc, AccountCustomEntity = Account, HostCustomEntity = Computer, IPCustomEntity
```